webui/react/src/utils/number.ts

```diff
@@ -25,3 +25,12 @@ export const roundToPrecision = (n: number, precision = 6): number => {
   const factor = 10 ** precision;
   return Math.round((n + Number.EPSILON) * factor) / factor;
 };
+
+export const findFactorOfNumber = (n: number): number[] => {
+  const abs = Math.abs(n);
+  const factors = [];
+  for (let i = 1; i <= abs; i++) {
+    if (abs % i === 0) factors.push(i);
+  }
+  return factors;
```

Comment:
**nit suggestion:** You can cut the loop down to half by figuring out what the other multiplier is when if you find one factor. So for example if you are evaluating 18 and you come across 3 as a factor, you also know 18 / 3 = 6 is the other factor pair right then instead of having to reach 18 % 6 === 0 portion of the loop.

Reply:
Do you think what I just pushed works?